plugin/Renderer/InputFieldTextRenderer.php

```diff
@@ -55,9 +59,21 @@ public function __construct($type, $name, $value = null)
 
 	public function render()
 	{
+		$textHtml = '';
+		if (!empty($this->getHint())) {
```

Comment:
Preferably `if ($this->getHint() !== '')`. `getHint()` and `setHint()` should get some `string` types as well.